.github/workflows/label_merge_conflicts.yml

```diff
@@ -0,0 +1,23 @@
+name: Manage labels
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types: [synchronize]
```

Comment:
Any reason not to run it on PR open as well, since people might PR something with merge conflicts in it already?

Reply:
If they already have merge conflicts, workflows won't run.

Reply:
Ah okay.